packages/hardhat/hardhat.config.ts

```diff
@@ -74,6 +74,8 @@ const config: HardhatUserConfig = {
       accounts: {
         accountsBalance: '1000000000000000000000000000'
       },
+      // IMPORTANT: SHOULD BE REMOVED AND FIXED BEFORE MERGING
```

Comment:
Easier way to fix this temp issue is to set the ```runs: 800``` under optimizer to 1 and see if that gets you under the limit and can raise it when we remove code.

Reply:
(spoiler, it does get us under the limit :) )

Reply:
Yep worked